pydarn/plotting/rtp.py

```diff
@@ -679,7 +699,7 @@ def plot_time_series(cls, dmap_data: List[dict],
     @classmethod
     def plot_summary(cls, dmap_data: List[dict], beam_num: int = 0,
                      groundscatter: bool = True, channel: int = 'all',
-                     coord: object = Coord.SLANT_RANGE, figsize: tuple = (11, 8.5),
+                     coord: object = Coords.SLANT_RANGE, hgt: float = 250.0, figsize: tuple = (11, 8.5),
```

Comment:
```  suggestion
                     coord: object = Coords.SLANT_RANGE, reflection_height: float = 250.0, figsize: tuple = (11, 8.5),
```

Any reason it is 250.0 here but at range time it's 250? Actually, you have it as an int there?